src/Domain/Creators/Cylinder.cpp

```diff
@@ -97,22 +97,44 @@ Cylinder::Cylinder(
               std::to_string(upper_bound_));
     }
   }
-  
-  // Expand initial refinement and number of grid points over all blocks
+
+  // Create block names and groups
   const size_t num_shells = 1 + radial_partitioning_.size();
   const size_t num_layers = 1 + height_partitioning_.size();
-  const size_t num_blocks = (1 + 4 * num_shells) * num_layers;
-  const ExpandOverBlocks<size_t, 3> expand_over_blocks{num_blocks};
+  static std::array<std::string, 4> direction_descriptions{"East", "North",
```

Comment:
if there is never a rotation, would `+x` `-x`, etc. be clearer? There is some undocumented definition of `East`, etc.

Reply:
The cardinal directions read nicer in the input file, so I added a section to the help string that explicitly defines all directions.

Reply:
sounds good :)